vendorconsent/tcf1/rangesection.go

```diff
@@ -88,7 +88,7 @@ func parseException(data consentMetadata, initialBit uint) (rangeException, uint
 }
 
 // parseUInt16  parses a 16-bit integer from the data array, starting at the given index
-func parseUInt16(data []byte, bitStartIndex uint) (uint16, error) {
+func parseUInt16(data []byte, bitStartIndex uint) (uint16, error) { // tiago: why not use the bitutils.ParseUint16 ?
```

Comment:
I believe this is here becasue bitutils was written when TCF2 was added, and not too much effort was put in to back porting it to the TCF1 code. Feel free to back port it in, on the other hand tcf1 should not be seeing any use anymore.

Reply:
We should consider removing TCF1 from this library now. That's well beyond the scope of this PR. I agree with @hhhjort, refactor it if you wish.